src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js

```diff
@@ -0,0 +1,876 @@
+import { Validators } from 'react-reactive-form';
+
+import FormComponents from '../../components/form';
+import IncidentNavigation from '../../components/IncidentNavigation';
+
+export default {
+  controls: {
+    custom_text: {
+      meta: {
+        label: 'Dit hebt u net ingevuld:',
+        type: 'citation',
+        value: '{incident.description}',
+        ignoreVisibility: true,
+      },
+      render: FormComponents.PlainText,
+    },
+
+
+
+    // woningdelen
+    extra_wonen_woningdelen_vermoeden: {
+      meta: {
+        ifOneOf: {
+          subcategory: [
+            'woningdelen',
+          ],
+        },
+        label: 'Weet u wat zich in deze woning afspeelt?',
+        subtitle: 'Vermoedens over bijvoorbeeld illegale activiteiten',
+        shortLabel: 'Vermoeden',
+        pathMerge: 'extra_properties',
+        className: 'col-sm-12 col-md-8',
+      },
+      render: FormComponents.TextInput,
+    },
+    extra_wonen_woningdelen_eigenaar: {
+      meta: {
+        ifOneOf: {
+          subcategory: [
+            'woningdelen',
+          ],
+        },
+        label: 'Weet u wie de eigenaar is van de woning?',
+        shortLabel: 'Eigenaar',
+        pathMerge: 'extra_properties',
+        className: 'col-sm-12 col-md-8',
+      },
+      options: {
+        validators: [Validators.required],
+      },
+      render: FormComponents.TextInput,
+    },
+    extra_wonen_woningdelen_adres_huurder: {
+      meta: {
+        ifOneOf: {
+          subcategory: [
+            'woningdelen',
+          ],
+        },
+        label: 'Weet u waar de officiële huurder woont?',
+        subtitle: 'De persoon die in de woning zou moeten wonen',
+        shortLabel: 'Adres huurder',
+        pathMerge: 'extra_properties',
+        className: 'col-sm-12 col-md-8',
+        values: {
+          zelfde_adres: 'Op het zelfde adres als de bewoners',
+          ander_adres: 'Op een ander adres dan de bewoners',
+          weet_ik_niet: 'Weet ik niet',
+        },
+      },
+      options: {
+        validators: [Validators.required],
+      },
+      render: FormComponents.RadioInput,
+    },
+    extra_wonen_woningdelen_aantal_personen: {
+      meta: {
+        ifOneOf: {
+          subcategory: [
+            'woningdelen',
+          ],
+        },
+        label: 'Hoeveel personen wonen er op dit adres?',
+        shortLabel: 'Aantal personen',
+        pathMerge: 'extra_properties',
+        className: 'col-sm-12 col-md-8',
+        values: {
+          een_persoon: '1 persoon',
+          twee_personen: '2 personen',
+          drie_personen: '3 personen',
+          vier_personen: '4 personen',
+          vijf_of_meer_personen: '5 of meer  personen',
+          weet_ik_niet: 'Weet ik niet',
+        },
+      },
+      options: {
+        validators: [Validators.required],
+      },
+      render: FormComponents.RadioInput,
+    },
+    extra_wonen_woningdelen_bewoners_familie: {
+      meta: {
+        ifAllOf: {
```

Comment:
What is the difference between `ifOneOf` and `ifAllOf`?

Reply:
ifOneOf: if 1 statement is true then the question will be shown
ifAllOf: if all statements are correct it will be true then the question will be shown